src/fiat-connect-client.ts

```diff
@@ -140,6 +145,20 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
         return handleError(data)
       }
 
+      const headerSetCookie = response.headers.get('set-cookie')
+
+      if (headerSetCookie) {
+        headerSetCookie.split(',').forEach(async (cookie) => {
+          await this.cookieJar.setCookie(
+            Cookie.parse(cookie) ?? '',
+            this.config.baseUrl,
+            {
+              ignoreError: true,
```

Comment:
What does `ignoreError` do here?

Reply:
The IgnoreError flag catches error during parsing and does nothing. I was messing with it during testing, but it shouldn't be there.